src/toil/cwl/cwltoil.py

```diff
@@ -48,6 +48,14 @@
 )
 from urllib import parse as urlparse
 
+
+# Warn user if the wrong cwltool version is installed.
+try:
+    from toil.cwl.utils import check_cwltool_version
+    check_cwltool_version()
+except ImportError:
+    raise
```

Comment:
Hmmm... this try/except looks like a no op.

Reply:
Ah... this can stop the linter from complaining about `PEP 8: E402 module level import not at top of file`. I think we have to do this before we import cwltool. Perhaps I should move this to `src/toil/cwl/__init__.py`?